.travis.yml

```diff
@@ -74,18 +70,14 @@ matrix:
       # start dependencies
       - ./scripts/start_apache_spark_cluster.sh "127.0.0.1" "7077" || travis_terminate 1
       - VIRTUAL_ENV=None IN_BACKGROUND=1 ./feature_extractor
-      # integration tests
-      - ./sbt "test-only * -- -n Integration"
       # hashing test repositories
       - ./hash src/test/resources/siva || travis_terminate 1
       # query without similarity
       - ./query ./src/test/resources/LICENSE
       # report without similarity
       - ./report
       # query with similarity
-      - docker cp ./src/test/resources/hashtables_dump.cql db:/hashtables_dump.cql
-      - docker exec -it db cqlsh -e "SOURCE '/hashtables_dump.cql'"
```

Comment:
No need for hashtables fixtures any more?

Reply:
No. Hash above produces hashtables now.